### DISCUSSION TITLE
Using PowerPlatform-DataverseServiceClient SDK in Azure Function V4 Isolated Process Model .NET 7

### DISCUSSION THREAD

**Discussion by twiga2013** [microsoft/PowerPlatform-DataverseServiceClient#365] [General]
Hi Matt

Any concerns or issue you see using the Azure Function V4 targeting .NET 7.0 in Isolated Process with Microsoft.PowerPlatform.Dataverse.Client SDK?

Thanks

**Comment by MattB-msft** [comment]
There are no known issues with it,  However we do not have a lot of experience with it testing wise. 
Our current focus is .net6 and .net8 previews.

**Reply by twiga2013** [reply]
Ok Thanks. I will wait until NET 8. 

I ran a quick POC have a singleton of ServiceClient on the second request an exception is been thrown this instance of ServiceClient has been disposed and cannot be used.
